specification/search/data-plane/readme.md

```diff
@@ -74,6 +76,6 @@ csharp:
   azure-arm: true
   license-header: MICROSOFT_MIT
   namespace: Microsoft.Azure.Search
-  output-folder: $(csharp-sdks-folder)/Search/DataPlane/Microsoft.Azure.Search/GeneratedSearchIndex
+  output-folder: $(csharp-sdks-folder)/Search/DataPlane/Microsoft.Azure.Search/Generated$(search-folder)

```

Comment:
@olydis Can you explain what this is going to do, or point me to docs I can read to understand all this new stuff?

Please note that we are using an old version of AutoRest from January and upgrading to the latest is not a priority for my team right now (in case that's relevant to this change).